<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ABHISHEK" failures="1" tests="2" name="Test" time="59.99" errors="0" timestamp="2022-07-14T20:35:12 IST">
  <testcase classname="testCases.RegisterAcctTest" name="registersuccess" time="8.423"/>
  <testcase classname="testCases.LoginTest" name="loginSuccess" time="2.589">
    <failure type="java.lang.AssertionError" message="expected [http://tutorialsninja.com/demo/index.php?route=account/account] but found [http://tutorialsninja.com/demo/index.php?route=account/login]">
      <![CDATA[java.lang.AssertionError: expected [http://tutorialsninja.com/demo/index.php?route=account/account] but found [http://tutorialsninja.com/demo/index.php?route=account/login]
at testCases.LoginTest.loginSuccess(LoginTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginSuccess -->
</testsuite> <!-- Test -->
